announcements/triage-wg-welcome.md

```diff
@@ -0,0 +1,24 @@
+The Release team is pleased to announce the **triage working group**, which will
+assume responsibility for triaging issues and pull requests in the
+[rust-lang](https://github.com/rust-lang) repositories as well as for
+diagnosing failures in Crater runs.
+
+Triaging is a high-impact activity that requires a relatively small time commitment.
+The goal is to prevent issues and pull requests from being ignored by the
+Rust team or forgotten by the contributors who submit them, and to ensure
+that everyone involved in the process knows what the next step is in the process
+toward resolving an issue or merging a pull request. In a large project
+like Rust, keeping up with community contributions is one of the main
+difficulties when scaling up development efforts, so the triage WG's efforts
+will alleviate a key pain point in Rust's growth.
+
+Triaging does not require reading or writing code. If you like the Rust project
+and would like to help it advance but do not feel comfortable submitting
```

Comment:
Well if you are comfortable with submitting PRs you can and should still join... 😂

Reply:
Hm, I didn't mean for that to sound exclusionary, but you're right, it can be taken that way. I've removed that paragraph and added the "not require...code" bit to its own sentence in the second paragraph. Thanks!